mlir/test/Conversion/ArithToEmitC/arith-to-emitc.mlir

```diff
@@ -173,3 +173,25 @@ func.func @arith_int_to_float_cast_ops(%arg0: i8, %arg1: i64) {
 
   return
 }
+
+// -----
+
```

Comment:
We did exhaustive testing for the cmpf predicates, so I guess we should do it here, too.
The interesting thing about the remaining operators (after eq and ult) for me is
1) that they use the right predicate in emitc
2) that they use the signed/unsigned.

So I think you can add a test case after this two extensive ones,
 that just checks those two properties (and not everything) like
```
func.func @arith_cmpi(%arg0: i32, %arg1: i32) -> i1 {
 %ule = arith.cmpi ule, %arg0, %arg1 : i32
  // CHECK: emitc.cmp le, {{.*}} : (ui32, ui32) -> i1
 %sle = arith.cmpi sle, %arg0, %arg1 : i32
  // CHECK: emitc.cmp le, {{.*}} : (i32, i32) -> i1
 %ne = arith.cmpi ne, %arg0, %arg1 : i32
  // CHECK: emitc.cmp ne, {{.*}} : (i32, i32) -> i1
...
}

```

Reply:
Done, I guess :)